docs/sw360-integration.md

```diff
@@ -0,0 +1,66 @@
+# SW360 integration
+
+This guide shows the possible ways to integrate SW360 in the ORT run. The order of the sections is the recommended 
+order in how to use the several ways.
+
+Since the [Eclipse project antenna](https://github.com/eclipse/antenna) is archived, it is also recommended using the 
+SW360 integration with ORT.
+
+## UploadResultToSw360Command
+
+The [`UploadResultToSw360Command`](../cli/src/main/kotlin/commands/UploadResultToSw360Command.kt) is a standalone ORT
+command, which takes an ORT result file as an input and creates components/releases in SW360 for the packages, 
+afterwards it creates the projects and links the created releases to the respective project.
```

Comment:
Starting from "afterwords", this can be a new sentence

Reply:
Is there a clear definition somewhere in the documentation of what a "project" here refers to? SW360 has a different understanding than ORT, if I am not mistaken.